source/beamline/BioXAS/BioXASFilterFlipper.cpp

```diff
@@ -0,0 +1,256 @@
+#include "BioXASFilterFlipper.h"
+
+BioXASFilterFlipper::BioXASFilterFlipper(const QString &deviceName, QObject *parent) :
+	BioXASBeamlineComponent(deviceName, parent)
+{
+	// Create class variables.
+
+	slides_ = 0;
+	filters_ = 0;
+	slideChangeStatus_ = 0;
+	runMode_ = 0;
+	nextStepTrigger_ = 0;
+	receiverStatus_ = 0;
+	receiverStageCWLimit_ = 0;
+	receiverStageCCWLimit_ = 0;
+	cartridgeStageCWLimit_ = 0;
+	cartridgeStageCCWLimit_ = 0;
+}
+
+BioXASFilterFlipper::~BioXASFilterFlipper()
+{
+
+}
+
+bool BioXASFilterFlipper::isConnected() const
+{
```

Comment:
I had a lot of issues with the indentation being a bit funny when appearing on github.
If you want to fix this in future:
- Go to Tools >> Options
- Select the **C++** list option
- Change **Align continuation lines**: to **With Spaces**
- Select the **Text Editor** list option
- Change **Align continuation lines**: to **With Spaces**

this should ensure that when continuing lines like this it appears correctly when viewing it on GitHub